controllers/ManagementController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace app\controllers;
+
+use app\controllers\mainController\MainController;
+use app\models\databaseModels\User;
+use Yii;
+use yii\data\ActiveDataProvider;
+use app\models\ModifyAccountForm;
+
+
+class ManagementController extends MainController
+{
+    /**
+     * Display a page where the accounts are shown
+     */
+    public function actionAccounts($id = null)
+    {
+        if(!is_null($id)) {
+            return $this->actionModifiyAccount($id);
+        }
+
+        $usersDataProvider = new ActiveDataProvider([
+            'query' => User::find(),
+            'pagination' => [
+                'pageSize' => 20,
+            ],
+        ]);
+
+        return $this->render('accounts', [
+            'usersDataProvider' => $usersDataProvider
+        ]);
+
+
+
+
+    }
+
+    private function actionModifiyAccount($id)
+    {
+        $user =  User::findOne(['id' => $id]);
+        //dd($user);
+        $form = new ModifyAccountForm();
+        if($form->load($_POST) && $form->validate()) {
```

Comment:
Why don't you use the ```Yii::$app->request->post()```

Reply:
Why though ? The two are equivalent.

Reply:
Yes, it is just to keep the framework logical. But it's not indispensable.